sample/OSCON17/sessions

```diff
@@ -0,0 +1,548 @@
+[{
+		"id": 1,
+		"title": "Wednesday opening welcome",
+		"subtitle": null,
+		"short_abstract": null,
+		"long_abstract": "Program chairs Rachel Roumeliotis and Kelsey Hightower open the first day of keynotes.",
+		"start_time": "2017-05-10T09:00:00",
+		"end_time": "2017-05-10T09:05:00",
+		"session_type": {
+
+		},
+		"track": {
```

Comment:
This event doesn't come under any track. There is a check for empty tracks in the code I believe, and so I don't think this will cause a crash. @iamareebjamal could you please confirm this?